source/Calamari/Kubernetes/KubernetesContextScriptWrapper.cs

```diff
@@ -48,11 +48,6 @@ public bool IsEnabled(ScriptSyntax syntax)
         {
             var workingDirectory = Path.GetDirectoryName(script.File);
 
-            if (environmentVars == null)
```

Comment:
Why remove this?

Reply:
Damn, I removed it from the wrong wrapper. Thanks for spotting this!